src/index.css

```diff
@@ -1,6 +1,33 @@
-@import 'https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css';
+@import 'https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css';
 
 body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+
+[data-bs-theme='light'] {
+  .btn-secondary {
+    --bs-btn-bg: --bs-body-color;
+    --bs-btn-hover-bg: unset;
+    --bs-btn-active-color: unset;
+    --bs-btn-active-bg: unset;
+    border: var(--bs-border-width) solid var(--bs-border-color);
+    color: black;
+  }
+}
+
+[data-bs-theme='dark'] {
+  .btn-secondary {
+    --bs-btn-bg: --bs-body-color;
+    border: var(--bs-border-width) solid var(--bs-border-color);
+  }
+}
+
+@media only screen and (max-width: 767px) {
```

Comment:
I added a breakpoint here to drop the top-line filters down into a column layout when the screen gets small. This should make it look fine on mobile and desktop